lua/rocks/operations/helpers.lua

```diff
@@ -29,11 +29,111 @@ local helpers = {}
 
 helpers.semaphore = nio.control.semaphore(1)
 
+---@class MultiMutRocksTomlWrapper
+---@field cache table<string, MultiMutRocksTomlWrapper> Cache for nested metatables
+---@field tables MutRocksTomlRef[] Rocks toml config metatables
```

Comment:
suggestion: if the paths are associated with the tables, let's make this a `table<string, MutRocksTomlRef>`.

Reply:
Did this at first but wanted to be able to control what order the tables are parsed in so that preference could be given to the base configuration (though, now I will change the order so that imports are given preference). From my understanding, the order isn't guaranteed when you use a string-keyed table. Any ideas on how I could give preference to a specific config file when matching the keys?

Reply:
Does the order still matter if we treat conflicts as an error? 
If so, we can make it a list of tuples:

```lua
---@class MutRocksTomlRefWithPath
---@field path string
---@field ref MutRocksTomlRef

-- ...
---@field tables MutRocksTomlRefWithPath[]
```

Reply:
Order won't matter if we don't support overrides and just treat conflicts as errors. Also, yeah sounds good on the tuple, just thought of this too today and I'm changing the implementation as we speak. I feel like it might be good to just implement with overrides in mind in the future because I think that will be the next logically request when config importing is supported.